engine/settings/base.py

```diff
@@ -233,6 +233,7 @@ class DatabaseTypes:
     "fcm_django",
     "django_dbconn_retry",
     "apps.phone_notifications",
+    "apps.zvonok"
```

Comment:
Could you please move enabling of zvonok app to line 685, where we plug-in apps required only for oss?